### ISSUE TITLE
Zerocoin lost after sending to basecoin

### ISSUE THREAD

**Issue by Veil-Grumpy** [Veil-Project/veil#368]
I sent 500 zerocoins to newly generated basecoin address. 
The transaction initially showed up as unconfirmed, and remained unconfirmed for about a day. 
In talking to the support team, they asked me to do a fresh install and to restore the walled from the seed words.
I did this as well as the usual commands e.g.  
                                                                                                                                                                                                               rescanblockchain
rescanzerocoinwallet
rescanringctwallet
searchdeterministiczerocoin 0 25000 2  

Once this was done, the uncofirmed transaction was no longer visible ,HOWEVER the original 500 was not returned to my balances. We tried reset wallet transactions, but again nothing. 

The 500 are going, not showing up anywhere, it's like they never existed.

TX: 5a943674a7fe461a90481a0efb1516211ca84ac3d94ebf53d6fac51b32fcd035

Veil Core version v1.0.1.4-3279c67fa (64-bit)

Issue from member: Toby#1551

**Comment by presstab** [comment]
There is a very small possibility, but it is worth checking, that your zerocoin blockchain database dropped some information about the transaction.

One thing to try is using one of the blockchain snapshots and reloading, then using `rescanzerocoinwallet`.

**Comment by CryptoToby** [comment]
Unfortunately, that did not solve it, as per the post above.

**Comment by presstab** [comment]
@CryptoToby that transaction above is not on the blockchain `5a943674a7fe461a90481a0efb1516211ca84ac3d94ebf53d6fac51b32fcd035`.

I would say to try using `abandontransaction 5a943674a7fe461a90481a0efb1516211ca84ac3d94ebf53d6fac51b32fcd035 true` and then use `rescanzerocoinwallet` again and see if that gives you any luck.

**Comment by presstab** [comment]
Still an issue? Or should this be closed?

**Comment by CryptoToby** [comment]
Sorry I missed the last message. I tried the abandontransaction at the time of the error, as per support's suggestion. It didn't work (can't recall what it gave back). I'm not sure if anyone has tried to replicate it, I'm quite certain that the transaction did not go through as I had tried to send an amount that was greater than the amounts I could send (i.e., I sent 500 total, but I only had denominations of 10 at the time, exceeding the 20 maximum). If you want more direct responses feel free to message me on discord. I'll try to check here more often though.
